app/src/main/java/gov/hhs/cdc/trustedintermediary/etor/order/Order.java

```diff
@@ -0,0 +1,113 @@
+package gov.hhs.cdc.trustedintermediary.etor.order;
+
+import gov.hhs.cdc.trustedintermediary.context.ApplicationContext;
+import gov.hhs.cdc.trustedintermediary.wrappers.Logger;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Order {
+    private String id;
+    private String destination;
+    private String createAt;
+    private String client;
+    private String body;
+    private OrderMessage orderMessage = new OrderMessage();
+    private final DateTimeFormatter dateTimeFormat =
+            DateTimeFormatter.ofPattern("MM-dd-yyyy HH:mm");
+    private final Logger LOGGER = ApplicationContext.getImplementation(Logger.class);
```

Comment:
```suggestion
    @Inject Logger logger;
```

Reply:
I'm working on it. It appears some tests fail when using the annotation. I'm looking into it.

Reply:
Caveat: I know nothing of what failures you were running into, and I know you changed your code to no longer call the logger anyway.

I wonder if it is because the `@Inject` works only for singletons.  `Order` isn't meant to be a singleton, so when you construct a new `Order`, the logger isn't injected into it.  I think that's working as intended for now.